### PR TITLE
cilium: rename --node-port-acceleration=none to =disabled

### PR DESCRIPTION
**Pull request by borkmann**
Be more consistent and rename to "disable" since this is common for
various other flags already.

Signed-off-by: Daniel Borkmann <daniel@iogearbox.net>

### CODE REVIEW

**Comment by borkmann** [comment]
test-me-please

**Review by joestringer — APPROVED** [review]

File: daemon/cmd/daemon_main.go [hunk 1]
```diff
@@ -520,7 +520,7 @@ func init() {
 	flags.Bool(option.NodePortBindProtection, true, "Reject application bind(2) requests to service ports in the NodePort range")
 	option.BindEnv(option.NodePortBindProtection)
 
-	flags.String(option.NodePortAcceleration, option.NodePortAccelerationNone, "BPF NodePort acceleration via XDP (\"native\", \"none\")")
+	flags.String(option.NodePortAcceleration, option.NodePortAccelerationDisabled, "BPF NodePort acceleration via XDP (\"native\", \"disabled\")")
```
Comment (brb) [inline daemon/cmd/daemon_main.go]: Nit: `fmt.Sprintf("BPF NodePort acceleration via XDP ... (%s, %s)", option.NodePortAcceleration{Native,Disabled})`

Comment (borkmann) [reply daemon/cmd/daemon_main.go]: Agree, will change.

**Review by tgraf — APPROVED** [review]

**Comment by borkmann** [comment]
test-me-please

**Pull request merged**